### DISCUSSION TITLE
23.11.22 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#75] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 아티스트 좋아요 관련 기능 테스트 하면서 오류 수정 완료
  - 알림 기능 연결 중
- 내일은 어떤 일을 하나요?
  -  알림 기능 연결 완료
  - PR 생성

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * entity 연관관계 수정
 
* 내일은 어떤 일을 하나요?
  * 테스트 수행

**Reply by Jeonwoojae** [reply]
- 오늘
  - 주문 작업 중 생긴 주문-음악 연관관계 수정
- 내일
  - 수정한 내용 기반 추가 테스트 예정

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - Spring Security가 적용된 컨트롤러 테스트코드 작성법 조사중..
        - 테스트에서 Authentication을 사용하기 위해 커스텀 애노테이션 작성(`@WithSecurityContext`)
- 내일은 어떤 일을 하나요?
    - 테스트 코드 작성

**Reply by minju412** [reply]
세션문제..?
